templates/shared_services/firewall/porter.yaml

```diff
@@ -44,59 +45,49 @@ parameters:
     type: string
     default: "W10=" # b64 for []
     description: "Network rule collection array"
-  - name: stateful_resources_locked
-    env: STATEFUL_RESOURCES_LOCKED
-    type: boolean
-    default: true
 
 mixins:
   - terraform:
-      clientVersion: 1.2.6
+      clientVersion: 1.3.6
 
 install:
   - terraform:
       description: "Deploy shared service"
-      input: false
```

Comment:
Why is this line being deleted?

Reply:
It's not a valid property per the new porter.yaml schema. All terraform commands have no input by default.